src/constants.ts

```diff
@@ -55,3 +55,6 @@ export const REGIME_DISPLAY_PRIORITY: Record<PrivacyRegime, number> = {
   GDPR: 10,
   Unknown: 100,
 };
+
+/** These TCF Purposes cannot be allowed to process under legitimate interest legal basis */
```

Comment:
nit: can we add a reference for this fact? I assume you got it from some IAB doc

Reply:
will do once.. i find it